examples/demo_project/master_service/app/migrations/0001_initial.py

```diff
@@ -0,0 +1,80 @@
+#  Copyright © 2021 Ingram Micro Inc. All rights reserved.
+
+from django.conf import settings
+import django.contrib.auth.validators
+from django.db import migrations, models
+import django.db.models.deletion
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        ('auth', '0012_alter_user_first_name_max_length'),
```

Comment:
This migration must be run after migrations in application auth because User model have foreign key to auth.Group. `0012_alter_user_first_name_max_length ` is last migration in `auth`. This dependency is created automatically by django.

I replaced it by `('auth', '0001_initial')` to make it more obviously.